### PR TITLE
[Doc] add-returned-Description-of-return-fields-and-examples

### PR DESCRIPTION
**Pull request by hellolilyliuyi**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md [hunk 1]
```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
...
+
+```plaintext
++-------------+---------------+----------------+---------------------+--------------------+--------+--------------+-------+--------------------+---------+----------------+---------------+---------------------+--------------------------+----------+------------+----------+
+| PartitionId | PartitionName | VisibleVersion | VisibleVersionTime  | VisibleVersionHash | State  | PartitionKey | Range | DistributionKey    | Buckets | ReplicationNum | StorageMedium | CooldownTime        | LastConsistencyCheckTime | DataSize | IsInMemory | RowCount |
++-------------+---------------+----------------+---------------------+--------------------+--------+--------------+-------+--------------------+---------+----------------+---------------+---------------------+--------------------------+----------+------------+----------+
+```
+
+| **Field**                | **Description**                                              |
+| ------------------------ | ------------------------------------------------------------ |
+| PartitionId              | The partition ID.                                            |
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:32]: -> The ID of the partition.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md [hunk 2]
```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
...
+| PartitionName            | The name of the partition.                                   |
+| VisibleVersion           | The version number of the last successful load transaction. The version number increases by 1 with each successful load transaction. |
+| VisibleVersionTime       | The timestamp of the last successful load transaction.       |
+| VisibleVersionHash       | The hash value for the version number of the last successful load transaction. |
+| State                    | The state of the partition. Fixed value: `Normal`.           |
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:37]: state -> status

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md [hunk 3]
```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
...
+| PartitionKey             | The partition key that consists of one or more partition columns. |
+| Range                    | The range of the partition, which is a right half-open interval. |
+| DistributionKey          | The bucket key of hash bucketing.                            |
+| Buckets                  | The number of buckets for the partition.                     |
+| ReplicationNum           | The number of replicas for a tablet in the partition.        |
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:42]: a tablet -> per tablet OR each tablet

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md [hunk 4]
```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
...
+| StorageMedium            | Data storage device. HHD means hard disk drives. SSD means Solid-state drives. |
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:43]: -> The type of storage medium used to store the data from the partition. The value `HHD` indicates hard disk drives, and the value `SSD` indicates solid-state drives.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md [hunk 5]
```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
...
+| CooldownTime             | cooling time for data in the partition. After the cooling time,  data in this partition on SSDs is automatically migrated to HDDs by StarRocks. |
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:44]: -> The cooling time duration

Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:44]: image
extra space

Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:44]: After the cooling time,  data in this partition on SSDs is automatically migrated to HDDs by StarRocks. -> The data stored on SSDs will be automatically migrated to HDDs after their storage duration on SSDs exceeds the specified cooling duration.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md [hunk 6]
```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
...
+| LastConsistencyCheckTime | The time of the last consistency check. NULL indicates no consistency check was performed. |
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:45]: Wrap the value "NULL" in code format.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md [hunk 7]
```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
...
+| DataSize                 | Data size of the partition.                                  |
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:46]: -> The size of data in the partition.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md [hunk 8]
```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
...
+| IsInMemory               | Whether all the partition data is stored in memory.          |
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:47]: -> Whether all data in the partition is stored in memory.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md [hunk 9]
```diff
@@ -15,18 +19,69 @@ SHOW [TEMPORARY] PARTITIONS FROM [db_name.]table_name [WHERE] [ORDER BY] [LIMIT]
...
+| RowCount                 | The number of data rows of the partition.                    |
+
+
 ## Examples
```
Comment (amber-create) [inline docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:51]: Modify the following sentences according to the comments for the Chinese PR.

File: docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md (hunk 5 above)
Comment (evelynzhaojie) [reply docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:44]: @hellolilyliuyi create table has a similar description: storage_cooldown_time: the storage cooldown time for a partition. If the storage medium is SSD, SSD is switched to HDD after the time specified by this parameter. Format: "yyyy-MM-dd HH:mm:ss".

Comment (evelynzhaojie) [reply docs/sql-reference/sql-statements/data-manipulation/SHOW PARTITIONS.md:44]: this is a time point, not a duration

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]


:heart_eyes: **pass** : 0 / 0 (0%)

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Comment by hellolilyliuyi** [comment]
@mergify backport branch-3.1 branch-3.0 branch-2.5

**Pull request merged**